integration/spark-common-test/src/test/scala/org/apache/carbondata/integration/spark/testsuite/timeseries/TestTimeSeriesCreateTable.scala

```diff
@@ -82,149 +92,209 @@ class TestTimeSeriesCreateTable extends QueryTest with BeforeAndAfterAll {
        """.stripMargin)
   }
 
+  override def afterEach(): Unit = {
+    dropDataMaps("mainTable", "agg1_second", "agg1_minute",
+      "agg1_hour", "agg1_day", "agg1_month", "agg1_year")
+  }
+
   test("test timeseries create table 1") {
     checkExistence(sql("DESCRIBE FORMATTED mainTable_agg0_second"), true, "maintable_agg0_second")
-    sql("drop datamap agg0_second on table mainTable")
+    sql("DROP DATAMAP agg0_second ON TABLE mainTable")
```

Comment:
why is this change necessary??

Reply:
The operation should be upper case and table name should be down case according to SQL syntax, Hive syntax also like this.